bool isEnabled
int8 index
float32 torque
---
bool success
